### ISSUE TITLE
ANR caused nativeRelease() call, Android 6 only

### ISSUE THREAD

**Issue by RomanRadkoMasslight** [twilio/video-quickstart-android#244]
### Description

Application is not responding when internet connection is lost.

### Steps to Reproduce

1. Start a call (dialer has accepted the call).
2. Disable internet connection (to emulate lost).
3. ANR dialog appears.

#### Code

ANR caused by long performing of the nativeRelease() on the main thread (Android 6 only)
It has been found by analyzing a method trace
```java
    synchronized void release() {
        ThreadChecker.checkIsValidThread(handler);

        if (nativeRoomDelegate != 0) {
            nativeRelease(nativeRoomDelegate); // --> blocks main thread
            nativeRoomDelegate = 0;
            mediaFactory.release();
        }
    }
```

#### Expected Behavior

Release the native RoomDelegate should be performed instantly.

#### Actual Behavior

Blocks main thread.

#### Reproduces how Often

100% Android 6 only (tested also on 4 and 7 - not reproducible).

#### Video Android SDK

1.3.9 (the same for 1.3.11)

#### Android API

23

#### Android Device

Android Emulator

**Comment by aaalaniz** [comment]
Hey @RomanRadkoMasslight 

Is this bug reproducible in the quickstart?

Thanks!

**Comment by aaalaniz** [comment]
I believe this bug was fixed as part of our 2.0.0 GA release. Please reopen if encounter the issue again @RomanRadkoMasslight. Thanks!
